### DISCUSSION TITLE
Notification applet improvement [Do not disturb mod]

### DISCUSSION THREAD

**Discussion by Hilyxx** [linuxmint/discussions#417] [Pull requests]
Hello Mint team
Notification applet is a good feature but limited when notifications are turned off. When "Do not disturb" mod is enabled, applet icon isn't updated and it's hard to know whether notifications are enabled or not.

I'm not a developer but I have updated your notification applet with this new feature. It works well and without issues. 
When notifications are disabled, a new icon replaces empty icon and users can click on the applet to restart notifications.

[Kazam_screencast_00000.webm](https://github.com/linuxmint/discussions/assets/21991448/fd69ac08-6204-400e-8b78-6448a7fe013f)

**Comment by rcalixte** [comment]
This is already implemented in the [inhibit@cinnamon.org](https://github.com/linuxmint/cinnamon/tree/9dcb820/files/usr/share/cinnamon/applets/inhibit%40cinnamon.org) applet. This specific functionality will be available in Cinnamon 6.2.

**Reply by Hilyxx** [reply]
But notification applet icon is still not updated on the panel and users need to use 2 applets for this small feature whereas implementation in the main applet would be more appropriate.
Inhibit applet is not installed by default during system installation and some users don't use this applet.

**Comment by igorsantos07** [comment]
It is confusing to have disabled notifications not in the notifications applet, but in the applet originally used to disable the screen turning off. People also disable notifications to focus, not only for presentations, so the feature should, at the bare minimum, be present on both the applets.

**Reply by rcalixte** [reply]
@Hilyxx accomplished this in [Notifications Enhanced](https://cinnamon-spices.linuxmint.com/applets/view/393). You can add this and remove the existing notification applet.
